src/sentry/api/endpoints/organization_activity.py

```diff
@@ -14,7 +14,7 @@ def get(self, request, organization, member):
                 organization=organization,
                 teams__in=OrganizationMemberTeam.objects.filter(
                     organizationmember=member,
-                ).values('team')
+                ).values_list('team')
```

Comment:
most of these are useless, but it was the original changes to try and disable ``.values()`` (which is expensive in Django 1.8)